src/components/m-datepicker/js/datepicker.js

```diff
@@ -1,179 +1,81 @@
 import { EVENTS, AXA_EVENTS } from '../../../js/ui-events';
 import on from '../../../js/on';
 import fire from '../../../js/fire';
-import DeviceStateObserver from '../../../js/device-state';
```

Comment:
@jackblackCH 
@LucaMele 
Is responsive awareness somewhere else?
Or is it removed?

Reply:
Yes, made with css media queries only.

Reply:
I see.
This approach works too, but was not the original intention.
Whatever the teams decides is okay with me.

Reply:
i guess we have to challange both settings and stick with one. For me is not a criteria for PR Stop.

Reply:
As long as the observer is used only to add a css class, you can achieve the same with a media query. Please correct, if I missed another thing.